tools/generate_og_images.py

```
@@ -0,0 +1,58 @@
+import os
+import sys
```

Ruff (0.8.2)

`sys` imported but unused

Remove unused import: `sys`

(F401)